en/components/expansion_panel.md

```diff
@@ -0,0 +1,281 @@
+    - `igx-expansion-panel-body` - stores the component body. This part is only visible when the component is expanded.
+
+In the below example, we can create a small collapsible component that hold a bit of information about the hummingbird:
+```html
+<!-- expansible-panel.component.html -->
+<igx-expansion-panel>
+    <igx-expansion-panel-header>
+        <igx-expansion-panel-title>
+            Hummingbird
+        </igx-expansion-panel-title>
+        <igx-expansion-panel-description>
+            One of the smallest birds
+        </igx-expansion-panel-description>
+    </igx-expansion-panel-header>
+    <igx-expansion-panel-body>
+        Hummingbirds are birds native to the Americas that constitute the family Trochilidae. They are among the smallest of birds, most species measuring 7.5–13 cm (3–5 in) in length. Indeed, the smallest extant bird species is a hummingbird, the 5 cm (2.0 in) bee hummingbird weighing less than 2.0 g (0.07 oz).
+        <a href='https://en.wikipedia.org/wiki/Hummingbird'>Read more</a>
+    </igx-expansion-panel-body>
+</igx-expansion-panel>
+
+```
+The `IgxExpansionPanelComponent` exposes the `.igx-expansion-panel` class which we can use to style the component:
+
+```css
+<!-- expansion-panel.component.css -->
+
+.igx-expansion-panel
+{
+    font-family: Verdana;
+}
+```
+
+You can see the results below: 
+<div class="sample-container loading" style="height: 200px;">
+    <iframe id="expansion-sample-1-sample" frameborder="0" seamless width="100%" height="100%" src="{environment:demosBaseUrl}/expansion-sample-1" onload="onSampleIframeContentLoaded(this);"></iframe>
+</div>
+<div>
+    <button data-localize="stackblitz" class="stackblitz-btn" data-iframe-id="expansion-sample-1-sample" data-demos-base-url="{environment:demosBaseUrl}">view on stackblitz</button>
+</div>
+
+### Binding to properties and events
+
+We can add some logic to our component to make it show/hide the description depending on the state of the panel. We can do this by binding the description to the controls `collapsed` property:
+
+```typescript
+// in expansion-panel.component.ts
+import { IgxExpansionPanelComponent } from 'igniteui-angular';
+@Component({
+    ...
+})
+export class ExpansionPanelComponent {
+    @ViewChild(IgxExpansionPanelComponent, {read: IgxExpansionPanelComponent})
+    public panel: IgxExpansionPanelComponent;
+    public readMore: string = 'https://en.wikipedia.org/wiki/Hummingbird';
+}
+```
+
+```html
+<!-- in expansion-component.component.html -->
+...
+    <igx-expansion-panel-header>
+        ...
+        <igx-expansion-panel-description *ngIf="panel.collapsed">
+            One of the smallest birds
+        </igx-expansion-panel-description>
+    </igx-expansion-panel-header>
+```
+
+This way, the description will only be display when the component is collapsed.
```

Comment:
Move this before the code snippet (html) -> Using the following code sample will show the description only when the component is in collapsed state.